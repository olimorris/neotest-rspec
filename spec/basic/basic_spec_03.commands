:1
:lua require("neotest").run.run()
