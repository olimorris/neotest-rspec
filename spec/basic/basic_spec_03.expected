./SPEC/BASIC/BASIC_SPEC.RB@@PASSED@@adds two numbers together
./SPEC/BASIC/BASIC_SPEC.RB@@PASSED@@subtracts two numbers together
